deploy/aws/cloudbeat-aws.yml

```diff
@@ -2,10 +2,12 @@ cloudbeat:
   type: cloudbeat/cis_aws
   access_key_id: ${AWS_ACCESS_KEY_ID:""}
   secret_access_key: ${AWS_SECRET_ACCESS_KEY:""}
+  default_region: eu-west-1
```

Comment:
this field is not part of our integration [configuration](https://github.com/elastic/integrations/blob/main/packages/cloud_security_posture/data_stream/findings/agent/stream/aws.yml.hbs), no?

Reply:
Yes it should not be part of the UI, this is useful for setting a specific region for testing until we'll have [multi-region support](https://github.com/elastic/cloudbeat/issues/592). Anyway reverted for now :)